From 9bb48a230fe1ab9e64598bdc38605b3705e3e0c9 Mon Sep 17 00:00:00 2001
From: Florian Bruhin <me@the-compiler.org>
Date: Tue, 2 Feb 2021 19:09:22 +0100
Subject: [PATCH] Set persistent cookie policy properly when creating profile

When initially creating a new profile, we want to set its cookie policy
properly. This fixes a regression introduced in 2c81271 / 5782123.

Fixes #6120

(cherry picked from commit 3a0e411210158b0ea589a50c6581127beaaa95ce)
---
 qutebrowser/browser/webengine/webenginesettings.py | 1 +
 1 file changed, 1 insertion(+)

--- a/qutebrowser/browser/webengine/webenginesettings.py
+++ b/qutebrowser/browser/webengine/webenginesettings.py
@@ -257,6 +257,7 @@
         self.set_http_headers()
         self.set_http_cache_size()
         self._set_hardcoded_settings()
+        self.set_persistent_cookie_policy()
         self.set_dictionary_language()
 
     def _set_hardcoded_settings(self):
